packages/expo-image/ios/EXImageView.m

```diff
@@ -4,9 +4,28 @@
 #import <React/RCTConvert.h>
 
 static NSString * const sourceUriKey = @"uri";
+static NSString * const sourceScaleKey = @"scale";
+
+@interface EXImageView ()
+
+@property (nonatomic, strong) NSDictionary *source;
+@property (nonatomic, assign) RCTResizeMode resizeMode;
+@property (nonatomic, assign) BOOL needsReload;
+
+@end
 
 @implementation EXImageView
 
+- (instancetype)init
+{
+  if (self = [super init]) {
+    _needsReload = NO;
+    _resizeMode = RCTResizeModeCover;
+    self.contentMode = (UIViewContentMode)_resizeMode;
```

Comment:
This may be just me, but I'm not particularly fond of just force casting that way. Do you think extracting this to a helper function `contentModeFromResizeMode` would be an overkill?

Reply:
I copied this from RN, but I do agree a helper function would be better. Will do 👍

Reply:
Maybe we could:
- either use a static value (as we do with `RCTResizeModeCover`) and put `self.contentMode = UIViewContentModeScaleAspectFill` right after the `_resizeMode` (and maybe write a comment that these two should be kept in sync)
- or use `[self setResizeMode:RCTResizeModeCover]` so that we don't have an initializing code that is somehow special?

Reply:
I've added a helper function which does the conversion. In fact I've added a `EXImageTypes` class which now contains the other enums and helper functions as well, so that the EXImageView class stays a bit cleaner 👍